extensions/data-plane/data-plane-client/src/test/java/org/eclipse/edc/connector/core/validator/JsonObjectValidatorRegistryImplTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.core.validator;
+
+import jakarta.json.JsonObject;
+import org.eclipse.edc.validator.spi.ValidationResult;
+import org.eclipse.edc.validator.spi.Validator;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static jakarta.json.Json.createObjectBuilder;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.edc.junit.assertions.AbstractResultAssert.assertThat;
+import static org.eclipse.edc.validator.spi.Violation.violation;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class JsonObjectValidatorRegistryImplTest {
```

Comment:
I think this test landed in the wrong module, at least the GitHub overview is showing it under `extensions/data-plane/data-plane-client` which doesn't seem right